RiotSharp.Test/appsettings.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "ApiKey": "API_KEY",
+  "TournamentApiKey": "TOURNAMENT_API_KEY"
```

Comment:
This needs to be integrated in appveyor.yaml:

https://ci.appveyor.com/project/BenFradet/riotsharp/build/2.6.0.295#L7

Atm, the appveyor script replaces the api keys: https://github.com/BenFradet/RiotSharp/blob/develop/appveyor.yml#L61